Week 2

BentonFraizer opened the pull request:
Выполнены задания второй недели. 
Необходима проверка наставника.

BentonFraizer committed:
Единицы измерения переведены в rem и %

BentonFraizer committed:
Заменены единицы измрения, сделан адаптив

Единицы измерения заменены снова, т.к. в первый раз я из поменял вручную
при помощи калькулятора. Сейчас при помощи функции.
Также свёрстаны текущие страницы сайта для разрешений 768px и 480px,
сохраняя адекватное состояние по ширине вплоть до 320px.

BentonFraizer committed:
Подправлен путь до карты секции контактов

BentonFraizer committed:
Подправлены пути до иконок социальных путей

BentonFraizer committed:
Иконки собраны в спрайт

b-ars07 commented:
1. отсутствует ховер = 
![image](https://user-images.githubusercontent.com/9081116/86339077-37f86a80-bc64-11ea-9f6e-3cf214286398.png)
2. 
![image](https://user-images.githubusercontent.com/9081116/86339258-6bd39000-bc64-11ea-8b50-3ec9ab68e9c9.png)
3. 
![image](https://user-images.githubusercontent.com/9081116/86339301-79891580-bc64-11ea-8ce1-3d177ed0f13d.png)
![image](https://user-images.githubusercontent.com/9081116/86339323-80b02380-bc64-11ea-899a-3230915a699b.png)
![image](https://user-images.githubusercontent.com/9081116/86339343-87d73180-bc64-11ea-9b02-5d0f12da1807.png)
![image](https://user-images.githubusercontent.com/9081116/86339368-8f96d600-bc64-11ea-9ea5-f37aef2fa9dc.png)

BentonFraizer committed:
Устранены замечания наставника (hover и 320px)

Добавлен hover-эффект на элменты меню в header а также на номер телефона
и телефонную трубку также только в header.
Поправлена вёрстка в части соответствия минимально-необходимого
разрешения в 320px.

b-ars07 commented:
1. нет стилизации ховера по макету - 
![image](https://user-images.githubusercontent.com/9081116/86406172-addfee80-bcc3-11ea-9f20-711de03b6872.png)
2.
